Tasks/Kudeiko/Task2/Task2.py

```diff
@@ -0,0 +1,118 @@
+num_time = {1: 'один', 2: 'два', 3: 'три',
+            4: 'четыре', 5: 'пять', 6: 'шесть',
+            7: 'семь', 8: 'восемь', 9: 'девять',
+            10: 'десять', 11: 'одиннадцать', 12: 'двенадцать',
+            13: 'тринадцать', 14: 'четырнадцать', 15: 'пятнадцать',
+            16: 'шестнадцать', 17: 'семнадцать', 18: 'восемнадцать',
+            19: 'девятнадцать', 20: 'двадцать', 30: 'тридцать',
+            40: 'сорок', 50: 'пятьдесят'}
+
+
+def hour_sklan(h: int):
+    if h == 1:
+        return 'первого'
+    elif h == 2:
+        return 'второго'
+    elif h == 3:
+        return f'{num_time[3][:2]}етьего'
+    elif h == 4:
+        return f'{num_time[4][:3]}вёртого'
+    elif h == 7:
+        return f'{num_time[7][:2]}дьмого'
+    elif h == 8:
+        return f'{num_time[8][:3]}ьмого'
+    else:
+        return f'{num_time[h][:-1]}ого'
```

Comment:
how much effort would it be to just write these values down in another dictionary? we would spend a little more memory but gain more efficiency in terms of runtime

Reply:
Initially, this was a separate dictionary. But i decided to give in to your challenge for the shortest dictionary.

P.S.  I remember your words that with dictionaries the speed of the program is higher